### DISCUSSION TITLE
How to disable the previous dates before the current date on calendar?

### DISCUSSION THREAD

**Discussion by prpanto** [unovue/shadcn-vue#708] [Q&A]
How to disable the previous dates before the current date on calendar?

**Comment by kikky7** [comment]
Did you try `min-date` prop with current date?

**Reply by prpanto** [reply]
Can you give an example?

**Reply by sadeghbarati** [reply]
https://www.shadcn-vue.com/docs/components/date-picker.html#form

Check the Form Code tab

use `:min-value="today(getLocalTimeZone())"` on `Calendar` or `DatePickerRoot` component

**Reply by prpanto** [reply]
what happen if it is range calendar?

**Reply by prpanto** [reply]
what is a Matcher?
